codes/data/korea/generate_KOR_processed.R

```diff
@@ -74,11 +79,18 @@ for (t in 1:nrow(testing)){
 df$testing_regime[df$date >= testing$date_start[t]]  <- t
 }
 
+# pos_cases_quarantine which has different policy intensities
+pos_cases_quarantine <- filter(pol, policy == "pos_cases_quarantine")
+df$pos_cases_quarantine <- 0
+for (q in 1:nrow(pos_cases_quarantine)){
+  df$pos_cases_quarantine[df$date >= pos_cases_quarantine$date_start[q]]  <- q/(nrow(pos_cases_quarantine))
```

Comment:
do you not want to use the `policy_intensity` encoded for `pop_case_quarantine` in `data_sources.gsheet`?